x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/ConnectorTemplateRegistryTests.java

```diff
@@ -125,131 +115,6 @@ public void testThatNonExistingComposableTemplatesAreAddedImmediately() throws E
         });
     }
 
-    public void testThatNonExistingComponentTemplatesAreAddedImmediately() throws Exception {
```

Comment:
Nice to see a lot of this test plumbing go away with the use of system indices 👍